linkedazuredeploy.json

```diff
@@ -4,10 +4,69 @@
     "parameters": {
         "functionAppName": {
             "type": "string"
+        },
+        "siteName": {
+            "type": "string"
+        },
+        "hostingPlanName": {
+            "type": "string"
+        },
+        "reactProject": {
+            "type": "string"
+        },
+        "repoURL": {
+            "type": "string"
+        }
+        ,"branch": {
+            "type": "string"
         }
     },
     "variables": {},
-    "resources": [],
+    "resources": [
+        {
+            "type": "Microsoft.Web/sites",
+            "apiVersion": "2020-06-01",
+            "name": "[parameters('siteName')]",
+            "location": "[resourceGroup().location]",
+            "dependsOn": [],
+            "properties": {
+                "serverFarmId": "[parameters('hostingPlanName')]"
+
+            },
+            "resources": [
+                {
+                    "apiVersion": "2018-11-01",
+                    "name": "appsettings",
+                    "type": "config",
```

Comment:
Why this can't be in the main template ?  I think adding listkeys in this may fail sometimes, that is why it was kept blank.

Reply:
This was moved to the linked arm template because the `listkeys` call was failing when it was in the main template. The UI needs access to the Azure Function code to make the API calls so we can't get around calling `listkeys`